### PR TITLE
e2e tests updated a bit

### PR DESCRIPTION
**Pull request by pondrejk**
stop relying on rhel7_contenthost, using rhel8 now, future parametrization made possible

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/ -m e2e

**Comment by Gauravtalreja1** [comment]
@pondrejk can you look into the PRT result?

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/ -m e2e

**Review by jyejare — CHANGES_REQUESTED** [review]
ACK pending passing PRT results.

File: tests/foreman/endtoend/test_api_endtoend.py [hunk 1]
```diff
@@ -1053,12 +1053,14 @@ def test_positive_find_admin_user(self):
 
     @pytest.mark.skip_if_not_set('libvirt')
     @pytest.mark.tier4
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_match('8')
```
Comment (lhellebr) [inline tests/foreman/endtoend/test_api_endtoend.py:1057]: Why only 8?

**Review by lhellebr — APPROVED** [review]

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/ -m e2e

**Comment by pondrejk** [comment]
Putting this on hold, so that #11669 gets in first

**Comment by Gauravtalreja1** [comment]
@pondrejk Mentioned PR is merged, causing merge conflicts, can you please resolve them and rerun PRT?

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/ -m e2e

**Review by ogajduse — APPROVED** [review]
ACK, pending @lhellebr's question about the RHEL version used.

File: tests/foreman/endtoend/test_api_endtoend.py (hunk 1 above)
Comment (JacobCallahan) [reply tests/foreman/endtoend/test_api_endtoend.py:1057]: ping @pondrejk

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/ -m e2e

File: tests/foreman/endtoend/test_api_endtoend.py (hunk 1 above)
Comment (pondrejk) [reply tests/foreman/endtoend/test_api_endtoend.py]: @lhellebr @JacobCallahan well, eventually I reverted back to 7, there is a lot to improve in these tests and I don't give up, but I'd like to merge this pr to get them passing again. I'm working on a follow-up pr to (a) allow running against multiple hosts (currently precluded by the way repos are enabled in these tests), (b) make it use new style of host registration (register function doesn't currently play well with org dict returned from cli helper), etc. etc.

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/endtoend/ -m e2e

**Comment by pondrejk** [comment]
ok so build 2865 passed:
```
15:18:41  + pytest tests/foreman/endtoend/ -m e2e --external-logging --junit-xml=sat-results.xml -o junit_suite_name=sat-result
15:18:53  ============================= test session starts ==============================
15:18:53  platform linux -- Python 3.11.3, pytest-7.4.0, pluggy-1.2.0
15:18:55  Mandatory Requirements are up to date.
15:18:55  Optional Requirements Mismatch: redis==4.6.0 sphinx-autoapi==2.1.1 pytest-cov==4.1.0 pre-commit==3.3.3 sphinx==7.0.1 flake8==6.0.0 manage==0.1.15
15:18:55  To update mismatched requirements, run the pytest command with '--upgrade-required-reqs' OR '--upgrade-all-reqs' option.
15:18:55  shared_function enabled - OFF - scope:  - storage: file
15:18:55  rootdir: /opt/app-root/src/robottelo
15:18:55  configfile: pyproject.toml
15:18:55  plugins: ibutsu-2.2.4, mock-3.11.1, reportportal-5.1.9, services-2.2.1, xdist-3.3.1
15:18:57  collected 10 items / 8 deselected / 2 selected
15:18:57  
15:35:35  tests/foreman/endtoend/test_api_endtoend.py .                            [ 50%]
15:43:33  tests/foreman/endtoend/test_cli_endtoend.py .                            [100%]
```

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**